src/stcal/ramp_fitting/src/slope_fitter.c

```diff
@@ -753,7 +818,8 @@ is_pix_in_list(struct pixel_ramp * pr)
     cols[0] = 173;
```

Comment:
I take it this hard coded values is just for testing ? Should it be removed now or does it not matter ?

Reply:
Yes.  The function `is_pix_in_list` is a debugging function.  I use it to validate various properties of a specific pixel being processed.  I normally keep debugging functions in the code, even though they are not called when the code is cleaned up for a PR.  I marked this functions as a debugging function.